File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/old/servoTest.java
package org.firstinspires.ftc.teamcode.old;

import com.qualcomm.robotcore.eventloop.opmode.Disabled;
import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
import com.qualcomm.robotcore.hardware.CRServo;

@TeleOp(name="servoTest", group="Linear OpMode")
@Disabled
public class servoTest extends LinearOpMode {
    @Override
    public void runOpMode() {
        CRServo a = hardwareMap.get(CRServo.class, "a");
        CRServo b = hardwareMap.get(CRServo.class, "b");
        CRServo c = hardwareMap.get(CRServo.class, "c");

        waitForStart();
        while (opModeIsActive()) {
            a.setPower(gamepad1.left_stick_x);
            b.setPower(-gamepad1.left_stick_x);
            c.setPower(gamepad1.right_stick_x);
        }
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/oldtests/ServoTest.kt
package org.firstinspires.ftc.teamcode.oldtests

import com.qualcomm.robotcore.eventloop.opmode.Disabled
import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode
import com.qualcomm.robotcore.eventloop.opmode.TeleOp
import com.qualcomm.robotcore.hardware.Servo
import org.firstinspires.ftc.teamcode.Constants.KICKARM_DOWN
import org.firstinspires.ftc.teamcode.Constants.KICKARM_RELEASE

@TeleOp(name = "servo test", group = "Linear OpMode")
@Disabled
class ServoTest : LinearOpMode() {
    override fun runOpMode() {
        val s = hardwareMap.get(Servo::class.java, "s")
        waitForStart()
        s.position = 0.5
        while (opModeIsActive()) {
            if (gamepad1.a) {s.position = KICKARM_RELEASE }
            else if (gamepad1.b) { s.position= KICKARM_DOWN }
        }
    }
}